### PR TITLE
fix(www): line-height in docs feedback

### LINKED ISSUE #19379: www: Feedback Widget's lineHeight too big
## Description

the linge height of text in the `feedback-widget` is to much compared to the other text on page, and also with the limited height of the textarea

### Steps to reproduce

- go to a docs page
- click on feedback
- enter some text in textarea (without newline)

### Expected result

should see more text at once 

### Actual result

very heigh line height. it looks that there are always 2 new lines has done. at least should be egual to the docs page text

### Screenshot

Bildschirmfoto 2019-11-08 um 20 14 48

### PR DESCRIPTION
**Pull request by thecodingaviator**
## Description
Fixes extra line-height in the docs feedback. I’ve added padding because none was present at the too and it destroyed the look of the text box. 

## Related Issues
Closes #19379

### CODE REVIEW

**Comment by m-allanson** [comment]
Thank you @thecodingaviator - I'm cc'ing @fk for thoughts on this 🙏

File: www/src/components/feedback-widget/feedback-form.js [hunk 1]
```diff
@@ -71,6 +71,8 @@ const textareaStyles = {
   height: `5.5rem`,
   mt: 1,
   mb: 4,
+  lh: `1.25rem`,
```
Comment (fk) [inline www/src/components/feedback-widget/feedback-form.js]: Oh wow I didn't know that `lh` works. Nice!
I think we should be able to use `dense` (matching `1.25rem`) from the theme tokens here though?

```
lh: `dense`
```

File: www/src/components/feedback-widget/feedback-form.js [hunk 2]
```diff
@@ -71,6 +71,8 @@ const textareaStyles = {
...
+  padding: `0.5rem 0.75rem`,
```
Comment (fk) [inline www/src/components/feedback-widget/feedback-form.js]: Similar to above, we should be able to do

```
px: 3,
py: 2,
```

here to use theme's `space` tokens. They are not properly documented yet for the use with `theme-ui` and the `sx` prop, but I think you'll work through that: https://www.gatsbyjs.org/guidelines/design-tokens/

**Comment by thecodingaviator** [comment]
@fk I've tried something, do you think the `lineHeight` would work like this?

File: www/src/components/feedback-widget/feedback-form.js [hunk 3]
```diff
@@ -71,6 +71,9 @@ const textareaStyles = {
   height: `5.5rem`,
   mt: 1,
   mb: 4,
+  px: 3,
+  py: 2,
+  lineHeight: dense,
```
Comment (thecodingaviator) [inline www/src/components/feedback-widget/feedback-form.js]:
```suggestion
  lineHeight: `dense`,
```
Should it be like this?

Comment (fk) [reply www/src/components/feedback-widget/feedback-form.js]: Yep (`dense` without the backticks would be a variable).
I'll check out on localhost now!

Comment (fk) [reply www/src/components/feedback-widget/feedback-form.js]: Got sidetracked, sorry! Will do what I promised earlier, and properly review now … expect this to be merged in ~10 minutes.

**Comment by thecodingaviator** [comment]
Thanks @fk 💙

**Pull request merged**